### PR TITLE
fixed behavior scrolling apple aab 2149

### PR DESCRIPTION
**Pull request by maksim-an-upplabs**
@tonytlwu @squallstar 
fixing scroll behavior widget app submission
ref https://github.com/Fliplet/fliplet-studio/issues/2149
apple-app

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -3224,6 +3224,16 @@ $('.browse-files').on('click', function (e) {
   });
 });
 
+// Scroll accordion tab to the top
+$(document).ready(function (){
```
Comment (squallstar) [inline js/interface.js]: why is `$(document).ready` required in this case?

Comment (maksim-an-upplabs) [reply js/interface.js]: It is not necessary but I used it to make sure that the document is fully loaded.
If it is not need, I will change it without $(document).ready.

Comment (maksim-an-upplabs) [reply js/interface.js]: fixed

**Comment by tonytlwu** [comment]
Please update this PR to use the recent PR description template.

**Comment by tonytlwu** [comment]
Please update this PR to use the recent PR description template.

**Comment by MaksymShokin** [comment]
@tonytlwu @squallstar There is an issue with this PR. Apparently this fix works only locally and does not work on staging because aab is called as a modal window and as separate iframe while scrolling works in the parent iframe. Therefore we propose other things we can do:
1. Is there is a way to access parent iframe to scroll it? Because now we get this security error.
iframe-issue

2. Use scrollIntoView property. Although it does not work ideally, but definitely better than now.  
scrollIntoView

3. Add separate scroll inside aab inframe (although it looks worse than the existing view).

**Comment by squallstar** [comment]
@MaksymShokin for *1* you want to emit an event to the parent window and then handle it as necessary. Widgets to this via `Fliplet.Studio.emit()`.

**Comment by MaksymShokin** [comment]
@squallstar Apple and android submission components are called from Studio. Should we then emit scroll event to Studio using Fliplet.Studio.emit() ?

**Comment by squallstar** [comment]
@MaksymShokin yes

**Pull request closed without merging**